tests/v2/actions/workloads/deployment/verify.go

```diff
@@ -34,3 +37,52 @@ func VerifyDeployment(steveClient *steveV1.Client, deployment *steveV1.SteveAPIO
 
 	return err
 }
+
+// VerifyRolloutHistoryDeployment is a helper to validate rollout history deployment
+func VerifyRolloutHistoryDeployment(client *rancher.Client, clusterID, namespaceName string, deploymentName string, expectedRevision string) error {
+	wranglerContext, err := client.WranglerContext.DownStreamClusterWranglerContext(clusterID)
```

Comment:
this might not work if we have a local cluster.. can you add a conditional clause for the wrangler context for local clusters please?

Reply:
Yes, I can